oceanmesh/read_fort14.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+
+def read_fort14(finame="fort.14", read_bou=False):
+    """
+    Reads an ADCIRC fort.14 mesh file.
+
+    Args:
+        finame (str): Name of the input file. Default is 'fort.14'.
+        read_bou (bool): If True, also reads the boundary data.
+
+    Returns:
+        tuple: EToV, VX, B, opedat, boudat, title
+    """
+    print("Reading fort.14 file")
+
+    with open(finame, 'r') as fid:
+        # Leer la primera línea con el título
+        title = fid.readline().strip()
+        print(title)
+
+        # Leer el número de nodos y elementos
+        msgline = fid.readline().strip()
+        N = list(map(int, msgline.split()))
+
+        # Leer nodos (número, posición, profundidad)
+        Val = np.loadtxt(fid, max_rows=N[1], dtype=float)
+
+        # Leer triangulación
+        idx = np.loadtxt(fid, max_rows=N[0], dtype=int)
+
+        # Ordenar los datos leídos
+        EToV = idx[:, 2:5]
```

Comment:
Could be good to check that EToV is converted to zero-based indexing as that's how it's used in Python.